vim_menutrans_merger.py

```diff
@@ -42,51 +45,54 @@ def extract_translated_message(template_file, translation_file):
 def replace_template_translation(template_file, translated_dict):
     """Replace the translations in template_file with the content in translated_dict"""
 
-    newContent = ""
+    new_content = ""
     # Encode with "latin1", because we don't care about the correctness of 
     # non-ASCII character.
-    with open(template_file, encoding='UTF-8') as f:
+    with open(template_file, encoding='latin1') as f:
         for line_number, line in enumerate(f):
             line = line.strip()
             if not line:
-                newContent += "\n"
+                new_content += "\n"
                 continue
 
             # Split a line by white spaces but not backslash-escaped spaces.
             new_word_list = re.findall(r"(?:\\ |[^ \t])+", line)
             match_index = [each for each in re.finditer(r"(?:\\ |[^ \t])+", line)]
-            hasTranslated = True
+            has_translated = True
             
             if len(new_word_list) > 1 and new_word_list[0] in MENUTRANS_COMMANDS:
                 if (new_word_list[1].lower() in translated_dict):
-                    line = line[:match_index[1].end() + 1] + translated_dict[new_word_list[1].lower()][3]
+                    line = line[:match_index[1].end() + 1] 
+                    + translated_dict[new_word_list[1].lower()][3]
                 else:
-                    hasTranslated = False
+                    has_translated = False
             elif len(new_word_list) > 1 and new_word_list[0].startswith("g:menutrans_"):
                 if (new_word_list[0].lower() in translated_dict):
-                    line = line[:match_index[0].end() + 1] + translated_dict[new_word_list[0].lower()][3]
+                    line = line[:match_index[0].end() + 1] 
+                    + translated_dict[new_word_list[0].lower()][3]
                 else:
-                    hasTranslated = False
+                    has_translated = False
             elif (len(new_word_list) > 1 and new_word_list[0] == "let" and 
                     new_word_list[1].startswith("g:menutrans_")):
                 if (new_word_list[1].lower() in translated_dict):
-                    line = line[:match_index[2].end() + 1] + translated_dict[new_word_list[1].lower()][3]
+                    line = line[:match_index[2].end() + 1] 
+                    + translated_dict[new_word_list[1].lower()][3]
                 else:
-                    hasTranslated = False
+                    has_translated = False
             
-            if not hasTranslated:
+            if not has_translated:
                 line = '" ' + line
 
-            newContent += str(line_number + 1) + line
-            newContent += "\n"
+            new_content += str(line_number) + line
```

Comment:
Why? You're still generating the line number in the final result, and then your final result will no longer be a valid Vim script.